features/lessons/src/main/res/layout/ss_reading_activity.xml

```diff
@@ -22,16 +22,19 @@
 
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:bind="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools">
+    xmlns:bind="http://schemas.android.com/apk/res-auto">
 
     <data>
-
         <variable
             name="viewModel"
             type="com.cryart.sabbathschool.lessons.ui.readings.SSReadingViewModel" />
     </data>
 
+    <FrameLayout
```

Comment:
Why did you wrap whole layout to FrameLayout?

Reply:
Because `android: fitsSystemWindows =" true "` would not work properly on `CoordinatorLayout`. I was looking for the cause, but did not find it. After searching on StackOverflow, it was instructed to wrap the entire layout in Linear or Frame and put this flag there. Now it is working.

If anyone knows how to adapt it to CoordinatorLayout, go ahead. I did not find any other working solution.

Reply:
I see. Let's approve it, we can find better solution if any later.

Reply:
I also think so, I haven't found a better solution to the problem yet ;-)
@TinasheMzondiwa  what do you think?